o8g/scripts/actions.py

```diff
@@ -337,6 +337,7 @@ def flip(card, x=0, y=0):
     card.alternate = ALTERNATE_POLITICAL if not card.alternate else ''
     return
   card.isFaceUp = not card.isFaceUp
+  notify("{} flips {}.".format(me,card))
```

Comment:
Yeah i misunderstood your trello card, thought the isFaceUp function didn't show anything ^^
i'll correct that